### ISSUE TITLE
Accessibility Not Work

### ISSUE THREAD

**Issue by Nai367** [EVLF/Cypher-Rat-Source-Code#55]
Redmi Not 11 accessibility Not work after restart Mobile.

![IMG_20220818_182303](https://user-images.githubusercontent.com/111510612/185396160-ce26fe05-9ea9-4402-979a-7a0d8ca48f23.jpg)

**Comment by rooseveltQ** [comment]
yes, it may be because there was an attack on the account, so just go to the center where you bought this phone and ask to restart the account, it will work 100%, and do not test it on your device

> Redmi Not 11 accessibility Not work after restart Mobile.
> 
> ![IMG_20220818_182303](https://user-images.githubusercontent.com/111510612/185396160-ce26fe05-9ea9-4402-979a-7a0d8ca48f23.jpg)
